packages/next/src/generators/init/generator.ts

```diff
@@ -87,6 +96,13 @@ export default async function initGenerator(
         path.join(project.sourceRoot, 'tsconfig.spec.json'),
     );
 
+    const morphTree = tsMorphTree(tree);
+
+    console.info(chalk.green`Attempting to add configuration for react-axe`);
+    addReactAxeConfigToApp(project, morphTree);
+    console.info(
+        chalk.green`continuing execution of next:init generator after attempting to add react-axe`,
+    );
```

Comment:
Leftover logs for dev?

Reply:
yeah I thought it would be helpful to know that in the case where react-axe doesn't get added (due to the new next dir structure). It's helpful to know that altough the config for react couldn't be added, it's not the reason why the init generator failed (if it fails)